.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
The problem with cjs that it's not always can be squashed and tree shaked. And output becomes a huge monster like webpack or browserify. So why we introduced rollup then?